src/smc-hub/package.json

```diff
@@ -35,7 +35,7 @@
     "node-cjsx": "^2.0.0",
     "node-uuid": "^1.4.2",
     "node-zendesk": "^1.3.0",
-    "passport": "^0.3.0",
+    "passport": "^0.4.0",
```

Comment:
I assume you did no testing?  This is hard to test, of course...

Reply:
no, but I will

Reply:
I setup the google passport on test.cocalc.com yesterday, so at least there is that...